index.js

```diff
@@ -121,7 +121,8 @@ const config = {
     "--no-sandbox",
     "--disable-setuid-sandbox",
     "--disable-gpu",
-    '--js-flags="--max_old_space_size=500"'
+    '--js-flags="--max_old_space_size=200"',
```

Comment:
Can we test this first? I'm worried 200MB might not be enough.

Reply:
Yep, testing it right now